### DISCUSSION TITLE
Instant loading

### DISCUSSION THREAD

**Discussion by elite-flacco** [squidfunk/mkdocs-material#4767] [Q&A]
I have the following in my mkdocs.yml, but it's still reloading the page.  Are there any other requirements for this to work? How can I debug this? Thank you!

  features:
    - navigation.instant
    - navigation.tracking
    - navigation.sections
    - navigation.prune
    - navigation.top

**Comment by squidfunk** [comment]
Please provide a self-contained reproducible example and attach it here as a zip.
